frontend/src/lib/components/Errors/types.ts

```diff
@@ -0,0 +1,54 @@
+export interface ErrorTrackingException {
+    stacktrace: ErrorTrackingRawStackTrace | ErrorTrackingResolvedStackTrace
+    module: string
+    type: string
+    value: string
+}
+
+interface ErrorTrackingRawStackTrace {
+    type: 'raw'
+    frames: ErrorTrackingStackFrame[]
```

Comment:
I think there's some confusion here - My understanding is that `raw` stack traces don't have an array of `ErrorTrackingStackFrame` (`Frame` in the cymbal codebase), the have an array of `RawFrame`s, each of which contains language specific data. 

When all is said and done, there should be almost no `raw` stack traces to display in the front-end - cymbal should process all stack traces, and always produce `resolved` ones, even if individual stack frames in the trace have `resolved: false`. The reason we have a lot of `raw` stack traces is because cymbal outright rejects python stacks, emitting the event to clickhouse without doing any processing on the input stack traces. I think we should only have nice display logic for "resolved" stack traces (they're the only ones we'll have e.g. frame context etc for anyway), and for `raw` frames, we should display the `$cymbal_errors` (we should do that in all cases probably), as well as some simplified json view of the raw input data. A stacktrace being `raw` indicates cymbal failed to even parse it, rather than e.g. that there was some frame-specific error or something (that only causes a specific frame to have `resolve: false`, not for the whole stack trace to be `raw`)

Reply:
Ok, updated this to basically ignore `frames` on raw stack traces for now and show the errors instead.